src/System.Runtime/tests/System/StringTests.cs

```diff
@@ -1076,7 +1078,8 @@ public static void GetEnumerator_Generic_Invalid()
         [InlineData("Hello", "", StringComparison.InvariantCulture, false)]
         [InlineData("", "Hello", StringComparison.InvariantCulture, false)]
         [InlineData("", "", StringComparison.InvariantCulture, true)]
-        [InlineData("123", 123, StringComparison.InvariantCultureIgnoreCase, false)] // Not a string
```

Comment:
While adding my test cases I noticed ```[InlineData("123", 123, StringComparison.InvariantCultureIgnoreCase, false)]``` here is a duplicate.
Whomever added this, meant to instead add their test case here for ```StringComparison.InvariantCulture```.